File: GTTS/GTTS.Web/ClientApp/src/app/models/INL.UnitLibraryService.Models/get-units-paged_param.ts


import { IGetUnitsPaged_Param } from './iget-units-paged_param';

export class GetUnitsPaged_Param implements IGetUnitsPaged_Param {
  
	public PageSize?: number;
	public PageNumber?: number;
	public SortDirection: string = "";
	public SortColumn: string = "";
	public CountryID?: number;
	public IsMainAgency?: boolean;
	public UnitMainAgencyID?: number;
	public IsActive?: boolean;
  
}

File: GTTS/INL.UnitLibraryService.Models/GetUnitsPaged_Param.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace INL.UnitLibraryService.Models
{
    public class GetUnitsPaged_Param : IGetUnitsPaged_Param
    {
        public int? PageSize { get; set; }
        public int? PageNumber { get; set; }
        public string SortDirection { get; set; }
        public string SortColumn { get; set; }
        public int? CountryID { get; set; }
        public bool? IsMainAgency { get; set; }
        public long? UnitMainAgencyID { get; set; }
        public bool? IsActive { get; set; }
    }
}
